react-stopwatch-component/src/stopwatch.jsx

```diff
@@ -0,0 +1,70 @@
+import React from 'react';
+
+class StopWatch extends React.Component {
+  constructor(props) {
+    super(props);
+    this.handleStart = this.handleStart.bind(this);
+    this.handleStop = this.handleStop.bind(this);
+    this.handleReset = this.handleReset.bind(this);
+    this.state = { start: false };
+    this.interval = null;
+    this.seconds = 0;
+  }
+
+  handleStart() {
+    const timeDisplay = document.querySelector('.time');
+    this.interval = setInterval(() => {
+      timeDisplay.textContent = this.seconds;
+      this.seconds++;
```

Comment:
You should have seconds stored in state and here you would increment the seconds stored in state by 1 using `setState`